Rivet/Functions/Get-MigrationFile.ps1

```diff
@@ -56,22 +56,44 @@ function Get-MigrationFile
             Write-Debug -Message $_ 
             if( (Test-Path -Path $_ -PathType Container) )
             {
-                Get-ChildItem -Path $_ -Filter '*_*.ps1'
+                Get-ChildItem -Path $_ -Filter '*.ps1'
             }
             elseif( (Test-Path -Path $_ -PathType Leaf) )
             {
                 Get-Item -Path $_
             }
         } | 
         ForEach-Object {
-            if( $_.BaseName -notmatch '^(\d{14})_(.+)' )
+            if( $_.BaseName -eq 'schema' )
+            {
+                $schemaContents = Get-Content -Path $_.FullName

```

Comment:
This should be reverted to what you had before. `Update-Database` will pass in the path to the schema.ps1 file to run. All you need to do here is detect that it's the schema.ps1 file and set the id/name.

Reply:
If I don't compare the contents of the schema file here, and the schema file happens to be only the default contents. Convert-FileToMigration will throw errors on how the migrations doesn't have any push or pop operations.

Reply:
Update `Convert-FileToMigration` to no longer write this error.